bin/truffleruby

```diff
@@ -113,8 +139,6 @@ do
         elif [ "${val}" = "-cp" ]; then
             CP="$CP:$2"
             shift
-        elif [ "${val}" = "-cmd" ]; then
-            print_command="true"
```

Comment:
Could we have a simpler logic like:
if graalvm; add it in java_args?
That doesn't need another array or iterating.